SkietbaanBE/SkietbaanBE/Controllers/TestController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SkietbaanBE.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/Test")]
```

Comment:
You can use `[Route("api/[controller]")]` to automatically use the name of the controller in the uri

Reply:
changed , doing new PR now